float32 current_speed
float32 target_speed
float32 timestamp
